rig-core/src/embeddings/builder.rs

```diff
@@ -73,6 +73,7 @@ impl<M: EmbeddingModel, T: Embeddable> EmbeddingsBuilder<M, T> {
 ///
 /// let model = openai_client.embedding_model(TEXT_EMBEDDING_ADA_002);
 ///
+/// # tokio_test::block_on(async {
```

Comment:
Remove the `tokio_test::block_on` for now, we'll make the docstring codeblocks runable in a subsequent PR